tools/dataset_converters/foggy_cityscapes.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import glob
+import os.path as osp
+
+import cityscapesscripts.helpers.labels as CSLabels
```

Comment:
Please provide the installation command.

Reply:
it is pip install cityscapesscripts